src/main/java/no/nav/foreldrepenger/selvbetjening/oppslag/OppslagTjeneste.java

```diff
@@ -42,7 +41,7 @@ public PersonFrontend hentPerson() {
     private Søkerinfo søkerinfo() {
         LOG.info("Henter søkerinfo");
         var info = new Søkerinfo(tilPersonFrontend(oppslag.hentPerson()), innsyn.hentArbeidsForhold());
-        LOG.info("Hentet søkerinfo for {} med {} arbeidsforhold OK", partialMask(info.søker().fnr()), info.arbeidsforhold().size());
+        LOG.info("Hentet søkerinfo for {} med {} arbeidsforhold OK", info.søker().fnr(), info.arbeidsforhold().size());
```

Comment:
Greit med value object for fnr, men her er for loggens del penest å hente value  fra value object og bruke partialMask() ?